Lib/uuid.py

```diff
@@ -1,8 +1,8 @@
 r"""UUID objects (universally unique identifiers) according to RFC 4122/9562.
 
 This module provides immutable UUID objects (class UUID) and the functions
-uuid1(), uuid3(), uuid4(), uuid5(), uuid6(), and uuid8() for generating
-version 1, 3, 4, 5, 6, and 8 UUIDs as specified in RFC 4122/9562.
+uuid{N}() for generating UUIDs version N as specified in RFC 4122/9562 for
+N = 1, 3, 4, 5, 6, 7, and 8.
```

Comment:
I don't think we necessarily need to enumerate them here, we can see them in the file.
```suggestion
uuidN() for generating UUIDs for version N as specified in RFC 4122/9562.
```

Reply:
Yeah, for the Python file docstring I think we can be a bit less formal

Reply:
I don't really like `uuidN` because it's not that visible (the combination of 'd' + capital letter is always hard to read for me in general, which is why I used `{N}`). OTOH, `{N}` looks like invalid syntax =/